etest/docker/container.py

```diff
@@ -49,6 +49,14 @@ def create(overlay: str, *args, **kwargs):
     return container
 
 
+def create_simple(*args, **kwargs):
```

Comment:
I remember adding an overloaded version for a cleaner front-end API, but it didnt get staged. We can discuss its working on the next commit.

Reply:
That sounds good.  I'll file an issue to review the Docker API we keep for etest as well since there's plenty of cleanup we can do here to keep things encapsulated.  Let's land this API as is for now and we can revisit.